packages/@aws-cdk/aws-ecs-patterns/test/fargate/load-balanced-fargate-service.test.ts

```diff
@@ -316,6 +316,40 @@ test('setting platform version', () => {
   });
 });
 
+test('setting ephemeral storage on NLB Fargate Service', () => {
```

Comment:
Same as above: Let's add in validation failure cases for below 21 and above 200.